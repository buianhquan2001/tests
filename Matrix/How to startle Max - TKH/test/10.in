31
1945
1118
1686
1669
1093
1286
1113
1947
1190
1080
1919
1907
1721
1668
1423
1638
1150
1138
1540
1664
1055
1881
1011
1393
1188
1826
1889
1579
1149
1665
1990
