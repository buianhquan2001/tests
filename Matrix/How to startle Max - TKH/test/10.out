791342885
96529190
478231202
692237555
702024570
564621209
337221988
425430888
558045873
137310635
975163927
480939518
381323326
95567880
916933157
583102241
855494361
239793603
161331274
148459427
117391315
727688173
37104964
188559988
742977445
513878197
666760983
39750449
798958398
114040082
147028839
